app/views/webhooks/detail.njk

```diff
@@ -0,0 +1,93 @@
+{% extends "layout.njk" %}
+
+{% block pageTitle %}
+  Webhook {{ webhook.callback_url }} - {{currentService.name}} {{ humanReadableEnvironment }} - GOV.UK Pay
+{% endblock %}
+
+{% block side_navigation %}
+  {% include "includes/side-navigation.njk" %}
+{% endblock %}
+
+{% block mainContent %}
+<div class="govuk-grid-column-two-thirds">
+  <h1 class="govuk-heading-l page-title" title="{{ webhook.callback_url }}">
+    {{ webhook.callback_url | truncate(40) }}
+    <strong class="govuk-tag govuk-tag--blue">{{ webhook.status }}</strong>
+  </h1>
+
+  <p class="govuk-body">{{ webhook.description }}</p>
+
+  <p class="govuk-body">
```

Comment:
Not sure what's in the design here, but seems like this is unlabelled. Should it have a heading like `Payment events`?